docs/kubernetes/kctlr-create-service-vs.rst

```diff
@@ -59,16 +59,16 @@ For example, if your Service looks like this:
 Your F5 resource ConfigMap might look like this:
 
 .. literalinclude:: /kubernetes/config_examples/f5-resource-vs-example.configmap.yaml
-   :lines: 17,9-15,23-34
+   :lines: 1-7,9-15,23-34
 
 :fonticon:`fa fa-download` :download:`f5-resource-vs-example.configmap.yaml </kubernetes/config_examples/f5-resource-vs-example.configmap.yaml>`
 
-
 The :code:`backend.servicePort` property in the F5 resource matches the :code:`ports.port` property in the Service definition.
+The :code:`porst.targetPort` setting is the Pod/Container port to which traffic will be sent. This setting allows the |kctlr| to relate the Pod Node Ports and Endpoints to the BIG-IP virtual server.
```

Comment:
porst -> ports

This is a hairy concept that I don't think I explained to you well. The portSpec defines a port and a targetPort. The port we care about is the 'port' field. It's not the port we'll use to add to the BIG-IP, that comes from the frontend, it's just used to tie all the information together. This sentence: "This setting allows the |kctlr| to relate the Pod Node Ports and Endpoints to the BIG-IP virtual server." should apply to ports.port.

The targetPort is the Pod port, if just using kube-proxy this would be the Pod port that this Service port sends traffic to; for instance, Service:80 -> Pod:8888.

Does that make sense? It's all ports, ports, ports, and more ports so want to be clear.